lib/ssl/src/ssl.erl

```diff
@@ -97,6 +97,7 @@ information about the supported standards see [ssl(6)](ssl_app.md).
          prepend_cipher_suites/2, 
          append_cipher_suites/2,
          signature_algs/2,
+         ec_curves/2,
          eccs/0, 
          eccs/1, 
```

Comment:
I had more discussion with Rickard and Dan and would need to discuss more with you:
- why adding `ec_curves/2` and not `eccs/3` taking (Version, Description)
- or a bit confusing compared to `eccs` but inline with `cipher_suites/2,3` eccs(Description, Version)
- do we need to schedule removal for `eccs` and `prf`? maybe deprecation is enough for longer period of time?

Reply:
Because eccs/0 and eccs/1 are broken. And adding  eccs/2 taking version and description is confusing in regards to other functions in ssl and changing the order to adhere to the other functions is confusing in regards to the old one. So I though it is less confusing to deprecate the broken one and make a new one that adheres to the rest of the ssl API functions. It is a utility function and I doubt it is one of the most used ones. I simply think it feels cleaner. I sort of mended the old ones to return the least surprising result.

Reply:
we discussed it more on fika.
I agree with everyone: @IngelaAndin, @rickard-green , @dgud :-) Everyone has valid arguments.

Reply:
This was solved by considering the eccs documentation to be a bug and new documentation has been added in OTP-26 and corresponding documentation is add by this PR for master. So we do not need to deprecate ecss. But prf  future is to be decided on OTP technical board.